bundle/manifests/backstage-operator.clusterserviceversion.yaml

```diff
@@ -37,7 +37,10 @@ spec:
       kind: Backstage
       name: backstages.rhdh.redhat.com
       version: v1alpha1
-  description: Operator to deploy Backstage on Kubernetes
+  description: |
+    Operator to deploy Backstage on Kubernetes
+
+    The telemetry data collection feature is enabled by default. Red Hat Developer Hub sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh
```

Comment:
this will need to be changed similar ot https://github.com/janus-idp/operator/pull/372/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R64 so we point to showcase, not RHDH.